### DISCUSSION TITLE
Renaming Arabic sky cultures

### DISCUSSION THREAD

**Discussion by kajaji** [Stellarium/stellarium#2212] [Ideas]
I received a suggestion from @sushoff to have the Arabic sky cultures appear in sequence as a group in the starlore list, similar to Babylonian, Chinese, Greek, and Western starlores.

For this, I suggest the following names: Arabic (Indigenous), Arabic (Lunar Mansions), Arabic (Arabian Peninsula), and Arabic (Al-Sufi).

**Comment by alex-w** [comment]
OK, I'll do it.

**Reply by alex-w** [reply]
Done: https://github.com/Stellarium/stellarium/commit/60542cfff046a59456216b1082cde21de08cba2e

**Reply by kajaji** [reply]
Thanks Alex

**Reply by sushoff** [reply]
thanks you two!
